src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -242,6 +258,95 @@ void _set_vprec_inst_mode(vprec_inst_mode mode) {
   }
 }
 
+/******************** VPREC HELPER FUNCTIONS *******************
+ * The following functions are used to set virtual precision,
+ * VPREC mode of operation and instrumentation mode.
+ ***************************************************************/
+
+int compute_absErr_vprec_binary32(bool isDenormal, void *context, int expDiff,
```

Comment:
These two functions, can be made one, by passing the mantissa size as an argument. It is the only difference between them both.

Reply:
I think that regrouping the two functions into a single one would obfuscate the code too much and render code maintainance more difficult.
I have identified a case that was never used and removed it, so this has simplified a bit the code.